holoviews/core/data/__init__.py

```diff
@@ -17,11 +17,13 @@
 from .multipath import MultiInterface         # noqa (API import)
 from .image import ImageInterface             # noqa (API import)
 
+default_datatype = 'dictionary'
 datatypes = ['dictionary', 'grid']
 
 try:
     import pandas as pd # noqa (Availability import)
     from .pandas import PandasInterface
+    default_datatype = 'dataframe'
```

Comment:
When converting from gridded to columnar data throughout the code it usually has to cast the data to a specific datatype. Various places in the code hardcoded ``['pandas', 'dictionary']`` in these places, defining a default_datatype avoids having to hardcode this all over the place.

Reply:
Shouldn't this be "default_columnar_datatype', then?  Or are there no cases where columnar data needs to be cast into some gridded data type?

Reply:
Columnar data cannot be cast to gridded data without some kind of aggregation occurring. So that's correct. Would still be okay with changing it to ``default_columnar_datatype``.